shared.csv

```diff
@@ -1,33 +1,57 @@
 name,quadrant,ring,isNew,description
-Kotlin,Languages & Frameworks,Trial,TRUE,
-TypeScript,Languages & Frameworks,Trial,TRUE,
-Spring Security,Languages & Frameworks,Trial,TRUE,
-Kibana,Tools,Trial,TRUE,
-Elasticsearch,Infrastructure Components,Trial,TRUE,
-Filebeat,Tools,Trial,TRUE,
-Yarn,Tools,Trial,TRUE,
-Docker,Infrastructure Components,Trial,TRUE,
-GitHub Actions,Services,Trial,TRUE,
-Make,Tools,Trial,TRUE,
-Express,Languages & Frameworks,Trial,TRUE,
-Node.js,Languages & Frameworks,Trial,TRUE,
-React,Languages & Frameworks,Trial,TRUE,
-Redux,Languages & Frameworks,Trial,TRUE,
-Gradle,Tools,Trial,TRUE,
-Nimbus,Languages & Frameworks,Trial,TRUE,
-Postgresql,Infrastructure Components,Trial,TRUE,
-GraphQL,Languages & Frameworks,Assess,TRUE,At the moment there is no use case for our application but we'd like to use it at some point
-React i18next,Languages & Frameworks,Trial,TRUE,
-kustomize,Tools,Trial,TRUE,
-Kubernetes,Infrastructure Components,Trial,TRUE,
-Google Cloud Build,Services,Trial,TRUE,
-NextJS,Languages & Frameworks,Hold,TRUE,Benefits don't outweigh the effort for the time being
-JUnit 5,Languages & Frameworks,Trial,TRUE,
-GreenMail,Tools,Trial,TRUE,
+OKRs,Techniques,Adopt,FALSE,"OKRs stands for Objectives and Key Results. At the beginning of every quarter we plan which Objectives we want to accomplish. After we have agreed on which Objectives to tackle, we assign different Key Results to them which represent concrete steps or actions on how to achieve set Objectives."
```

Comment:
@alex13053 I would change to lower case (when not explaining the acronym) because upper-casing suggests a non-literal use of a word. Objective and key result are used in their literal meaning, though - that's also why we could shorten what a key result is - just a "key result" :slightly_smiling_face: We could also add how we work with the OKRs during the quarter and what we use them for.

```suggestion
OKRs,Techniques,Adopt,FALSE,"OKRs stands for Objectives and Key Results. At the beginning of every quarter we plan which objectives we want to accomplish. Each objective is broken down into key results. Our team OKRs are in line with the company OKRs. Every week we update our OKR sheet with a value between 0 and 10 for each key result, to express how likely we find it to complete that key result. That way we can always judge our current progress."
```